src/components/hooks/useContributionsReminderEmailForm.ts

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import { isValidEmail } from '../modules/utils/reminders';
+
+type SubmitHandler = (e: React.FormEvent<HTMLFormElement>) => void;
+
+interface ContributionsReminderEmailForm {
+    email: string;
+    inputError?: string;
+    updateEmail: (e: React.ChangeEvent<HTMLInputElement>) => void;
+    handleSubmit: (onSubmit: () => void) => SubmitHandler;
+}
+
+export function useContributionsReminderEmailForm(): ContributionsReminderEmailForm {
```

Comment:
I thought about making one big hook that combines this one and `useContributionsReminder`, but this felt a little cleaner

Reply:
I think that makes sense.
This one maintains the state of the email field + validation.
The other one maintains the state of the reminder signup request.